app/models/family.rb

```diff
@@ -1,3 +1,5 @@
 class Family < ApplicationRecord
   validates_presence_of :last_name, :capability
+
+  scope :by_last_name, -> { order("lower(last_name)") }
```

Comment:
Can we still follow the style guidelines: `def self.by_last_name`?

Reply:
Done in sha `7470b57`